File: 프로그래머스/lv2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include <string>
#include <vector>
#include <iostream>
using namespace std;

string solution(string s) {
    string answer = "";
    int max_num , min_num, first =0;
    for(int i=0; i< s.length(); i++)
    {
        string num ="";
        while( i< s.length() &&  s[i] !=' ' )
        {
            num +=s[i];
            i++;
        }       
        if( first == 0)
        {
            first = 1;
            max_num = stoi(num);
            min_num = stoi(num);
        }
        else
        {
            if( max_num <= stoi(num))max_num = stoi(num);
            if( min_num >= stoi(num))min_num = stoi(num);
        }
    }
    answer = to_string(min_num) + ' ' + to_string(max_num);
    return answer;
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String[] answer = s.split(" ");
        
        int[] nums = new int[answer.length];
            
        int minNum,maxNum;
        minNum = maxNum = Integer.parseInt(answer[0]);
        for ( int i=1; i< answer.length;i++) {
            nums[i] = Integer.parseInt(answer[i]);
            if(minNum > nums[i]) minNum =nums[i];
            else if (maxNum < nums[i] ) maxNum = nums[i];
        }
        
        return minNum + " " + maxNum;
    }
}